volume/store/store.go

```diff
@@ -194,12 +194,14 @@ func (s *VolumeStore) create(name, driverName string, opts map[string]string) (v
 		}
 	}
 
-	logrus.Debugf("Registering new volume reference: driver %q, name %q", driverName, name)
```

Comment:
I did not think log drivername = "" is good, as whether or not, you would use local driver name for that, in that case,

why not tell we are using drivername is "local".

If we like to keep "" old log message,(which I think it is misleading)